base-notebook/Dockerfile.ppc64le

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) IBM Corporation 2016 
```

Comment:
Well, I have found it easier to keep the full Dockerfile.ppc64le handy, to generate the patchfile like this: "diff -Naur Dockerfile Dockerfile.ppc64le > Dockerfile.ppc64le.patch" if we need to create a new patch file. Also it is good to have the full Dockerfile as a reference that we can quickly link to.